src/index.html

```diff
@@ -16,17 +16,28 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-wrapper">
+    <form
+      class="search-container"
+      data-qa="big"
+    >
+      <input
+        class="search-bar search-bar__big"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form
+      class="search-container"
+      data-qa="small"
+    >
+      <input
+        class="search-bar search-bar__small"
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
Similarly to the previous comment, if 'search-bar__small' is a modifier, use 'search-bar search-bar--small' to follow BEM naming conventions properly.